content/blog/knative-eventing-for-observability.md

```diff
@@ -0,0 +1,456 @@
+---
+title: "Using Knative Eventing for Better Observability"
+featured: true
+weight: 1
+description: >
+  Getting a better picture of Kubernetes events
+date: 2021-02-22
+topics:
+  - Kubernetes
+tags:
+  - Kubernetes
+  - API
+  - Development
+  - Knative
+patterns:
+  - Deployment
+# Author(s)
+team:
+  - Tyler Britten
+---
+
+{{< youtube id="9lBaKKe-59E" class="youtube-video-shortcode" >}}
```

Comment:
Maybe end with this or put it in the middle somewhere? Feels weird opening a blog with a video.

Reply:
I like these at the start (with a description saying something like "watch this if you prefer video to blog text vomit" ) for accessibility to the many people who learn better from video than reading.